app/src/main/java/com/course/week1/activity/ContentActivity.kt

```diff
@@ -0,0 +1,49 @@
+package com.course.week1.activity
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.course.week1.R
+import com.course.week1.adapter.ContentAdapter
+import kotlinx.android.synthetic.main.activity_content.*
+
+class ContentActivity : AppCompatActivity() {
+
+    companion object {
+        const val REQUEST_CODE_CONTENT_RESULT = 1001
+        const val CONTENT_ITEM_KEY = "contentItemKey"
+    }
+
+    private val contentAdapter = ContentAdapter()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_content)
+
+        rvContent.layoutManager = LinearLayoutManager(this, RecyclerView.VERTICAL, false)
+        rvContent.adapter = contentAdapter
+
+        initContentList()
+        contentAdapter.onItemClick = ::onItemClick
+    }
+
+    private fun initContentList() {
+        val array = mutableListOf<String>()
```

Comment:
Более краткая версия
```suggestion
        val stringArray = resources.getStringArray(R.array.contentArray)
        val array = listOf<String>(*stringArray)
        contentAdapter.setData(array)
```